XboxKeyboardMouse/XboxController/Emulation/XboxStream.cs

```diff
@@ -28,8 +29,54 @@ public struct RECT {
 
         public static Thread tMouseMovement;
 
+        private static readonly string[] defaultAppNames = {
```

Comment:
Could you add the Playstation remote streaming too?  I believe we could get this app to work with Playstation.

Reply:
Filed #161